docs/guides/permissions.md

```diff
@@ -4,4 +4,73 @@ title: Permissions
 
 # Permissions
 
-Documentation coming soon
+Permissions can be managed using `Permission` classes. A `Permission` class extends `BasePermission` and has a `has_permission` method. It can be hooked up to a field using the `permission_classes` keyword argument. A simple example looks like this:
```

Comment:
```suggestion
Permissions can be managed using `Permission` classes. A `Permission` class extends `BasePermission` and has a `has_permission` method. It can be added to a field using the `permission_classes` keyword argument. A simple example looks like this:
```